mails/themes/classic/core/guest_to_customer.html.twig

```diff
@@ -27,7 +27,8 @@
 {% endif %}
             <span>
               {{ 'Your guest account for [1]{shop_name}[/1] has been transformed into a customer account.'|trans({'[1]': '<span><strong>', '[/1]': '</strong></span>'}, 'Emails.Body', locale)|raw }} <br/><br/>
-              <span><strong>{{ 'E-mail address:'|trans({}, 'Emails.Body', locale)|raw }}</strong></span> {email}
+              <span><strong>{{ 'E-mail address:'|trans({}, 'Emails.Body', locale)|raw }}</strong></span> {email}<br/><br/>
+              {{ 'To set up your password, please use the following link:'|trans({}, 'Emails.Body', locale)|raw }} <br/> <a href="{url}">{url}</a>
```

Comment:
@LouiseBonnard I improvised some wording here, don't hesitate if you have a better idea ;)

Reply:
```suggestion
              {{ 'Click on the following link to set up your password:'|trans({}, 'Emails.Body', locale)|raw }} <br/> <a href="{url}">{url}</a>
```